pages/common/wget.md

```diff
@@ -5,24 +5,28 @@
 
 - Download the contents of an URL to a file:
 
-`wget -O {{filename}} {{url}}`
+`wget {{https://example.com}}`
 
-- Limit download speed:
+- Download a single web page and all its associated resources:
 
-`wget --limit-rate={{200k}} {{url}}`
+`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
 
-- Continue an incomplete download:
+- Download a full website, with 3-second intervals between requests:
```

Comment:
The original example had a "target_folder" too. Are you sure we don't need it anymore ?

Reply:
I've replied below in the thread corresponding to the removed line.